File: benchmark/compare/standard/fibonacci.lua
function fibonacci(n)
    if n <= 1 then return n end
    return fibonacci(n - 1) + fibonacci(n - 2)
end

_ = fibonacci(10)
local start = os.clock()
local result = fibonacci(30)
local elapsed = (os.clock() - start) * 1000
print(string.format("fibonacci(30) = %d", result))
print(string.format("elapsed_ms=%.2f", elapsed))

File: benchmark/compare/standard/fibonacci.py
import time

def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)

_ = fibonacci(10)
start = time.perf_counter()
result = fibonacci(30)
elapsed = (time.perf_counter() - start) * 1000
print(f"fibonacci(30) = {result}")
print(f"elapsed_ms={elapsed:.2f}")
